main.go

```diff
@@ -159,53 +163,61 @@ func before(c *cli.Context) error {
 
 func start(c *cli.Context) error {
 	names := c.Args()
+        if c.GlobalBool("oneshot") {
+		log.SetLevel(log.DebugLevel)
```

Comment:
We shouldn't fix here to debug-level, we should also honor the `--debug` flag in this case. The debug flag is used in the `before` method, so you could simply drop `log.SetLevel(log.DebugLevel)` here.